docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -1,44 +1,49 @@
 ---
-title: Code Builder Quick Start
+title: Get Started with Code Builder
 lang: en
 ---
 
 ## Overview
+
 Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment. Before you start, see [Code Builder Overview](./en/codebuilder/about) for more information and important considerations.
 
 ### Don’t Forget to Save Your Work
-Working in the cloud has its advantages. However, unlike working on a desktop where you save your files to a local machine, you must either deploy your changes to your org, or commit them to source control to save your work. Save your work before you close the Code Builder tab on your browser so you don’t lose it. 
```

Comment:
@sbudhirajadoc - Suggested change:
Working in the cloud has its advantages. However, unlike working on a desktop where you save your files to a local machine, you must either deploy your changes to your org, or commit them to source control to save your work somewhere permanent.  Work saved in your Code Builder environment will be there when you come back but Code Builder is not suited for permanent-long-term storage.  We recommended saving your work before you close the Code Builder tab.